components/options_resolver.rst

```diff
@@ -96,7 +96,7 @@ the ``Mailer`` class makes a mistake?
 .. code-block:: php
 
     $mailer = new Mailer(array(
-        'usernme' => 'johndoe',
+        'usernme' => 'johndoe',  # usernAme misspelled 
```

Comment:
can you please use `//` instead of `#` for PHP comments?

Other than that, +1 for this change.